src/MICore/Transports/LocalLinuxTransport.cs

```diff
@@ -17,11 +17,17 @@ public class LocalLinuxTransport : StreamTransport
         private const string SudoPath = "/usr/bin/sudo";
         private const string GnomeTerminalPath = "/usr/bin/gnome-terminal";
         private const string XTermPath = "/usr/bin/xterm";
-        private const string FifoPrefix = "cpp-debug-fifo-";
+        private const string FifoPrefix = "Microsoft-MIEngine-fifo-";
 
         private string _gdbStdInName;
         private string _gdbStdOutName;
         private FileSystemWatcher _fifoWatcher;
+        private readonly MIMode _miMode;
+
+        public LocalLinuxTransport(MIMode miMode)
```

Comment:
I don't think this constructor is necessary - the value is only used in InitStreams, and we can get it from localLaunchOptions.DebuggerMIMode instead of having it passed in at instantiation.